Login/login.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <title>BuddyUp Login</title>
+
+        <!-- Include links to CSS stylesheets below -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.5/css/bulma.min.css" />
+        <link rel="stylesheet" href="styles.css" />
+    </head>
+
+    <body>
+        <div id="root">
+            <div id="header">
+                    <nav class="level" id="links">
+                            <p class="level-item has-text-centered">
+                                    <img src="https://bulma.io/images/bulma-type.png" alt="" style="height: 30px;">
+                                  </p>
+                            <p class="level-item has-text-centered">
+                              <a class="link is-info">Home</a>
+                            </p>
+                            <p class="level-item has-text-centered">
+                              <a class="link is-info">About</a>
+                            </p>
+                            <p class="level-item has-text-centered">
+                              <a class="link is-info">Trips</a>
+                            </p>
+                          </nav>
+            </div>
+
+        </div>
```

Comment:
the `<div>` with root id should cover the whole html body, so you want to move this closing tag to the end of the body. otherwise the rest of the file is unreachable with the $('#root') jquery statement.